test/functional/page_objects/visualize_page.js

```diff
@@ -363,6 +363,13 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       await PageObjects.header.waitUntilLoadingHasFinished();
     }
 
+    async clickUnlinkSavedSearch() {
+      const unlinkSavedSearch = await testSubjects.find('unlinkSavedSearch');
+      await remote.moveMouseTo(unlinkSavedSearch);
+      await remote.doubleClick();
```

Comment:
can you move this into the find and testSubjects service as a new doubleClick function?  It will then verify the element doesn't become stale between the find and the click and encapsulate it in a retry.